docs/power_consumption.md

```diff
@@ -0,0 +1,67 @@
+# Power Consumption
+
+## Intro
+
+We decided to take a closer look at the occurrence of energy consumption on apu
+platforms. Measuring energy is valuable if we strive to improve firmware.
+Thanks to this information, we are also able to detect errors quickly.
+
+## Setup testing enviroment
+
+Measuring power consumption is possible thanks to the connection of RTE with the
+[INA219](https://download.kamami.pl/p564714-ina219.pdf) sensor.RTE is a tool
+which helps debugging and testing devices.The INA219 is a current shunt and
+power monitor.Connect RTE I2C header (J9) with INA219 I2C header to read from 
+the sensor.
+
+ RTE I2C header (J9)        | INA219 I2C header
+:--------------------------:|:---------------------------:
+ 1 (3V3)                    | (Vcc)
+ 2 (SCK)                    | (SCL)
+ 3 (SDA)                    | (SDA)
+ 4 (GND)                    | (GND)
+
+Then you have to cut one of the wires in the power cable going from the RTE to
+the device. If we cut the plus, the part from RTE is connected to Vin+, and the
+part from the device to Vin-. If we cut the minus, we have to do the opposite.
```

Comment:
So which way should somebody due this, and why?    (It seems semi-obvious that high-side measurement is better if possible, and that seems to be part of the point of the INA219, that it can do this.)

Reply:
Changed so that there is no ambiguity.